bundle-size/github-utils.js

```diff
@@ -40,7 +40,7 @@ function getBuildArtifactsFileParams_(filename) {
 }
 
 /**
- * Utils for GitHub actions.
+ * Utils for GitHub actions that are performed by the user based token.
```

Comment:
Nit: "performed with a user-authenticated token" or something similar is a bit clearer to people who aren't as steeped in the types of tokens. A link to docs with the two types, here or somewhere else, might be useful.

Reply:
Done